src/components/SearchBar.tsx

```diff
@@ -31,61 +33,65 @@ type SearchBarCommandsType = {
   cancelSearch: (viewRef: React.ElementRef<typeof NativeSearchBar>) => void;
 };
 
-class SearchBar extends React.Component<SearchBarProps> {
-  nativeSearchBarRef: React.RefObject<SearchBarCommands>;
+const SearchBar = React.forwardRef<SearchBarCommands, SearchBarProps>(
+  function SearchBar(props, ref) {
+    const nativeSearchBarRef = React.useRef<SearchBarCommands | null>(null!);
+    React.useImperativeHandle(ref, () => ({
```

Comment:
Nitpick: I wonder whether we can create `SearchBarInner` and then just write 

```ts
const SearchBar = React.forwardRef<SearchBarCommands, SearchBarProps>(SearchBarInner);
```

to avoid whole level of indentation?

Reply:
We could also do it for `Screen` in separate PR.

Reply:
Taking into consideration that there already exists `InnerScreen` we could use `Impl` suffix or something 😅

Reply:
[615eddf](https://github.com/software-mansion/react-native-screens/pull/2132/commits/615eddf4a654e4d97c8f99733ba962ae3d0b39af)

Reply:
https://github.com/software-mansion/react-native-screens/pull/2132/commits/351a18702deb8873ad48cd9cec3686c29c26741c